syntax = "proto3";

package bot_admin;

option go_package = "genproto/bot_admin";

import "google/protobuf/empty.proto";

// Botni boshqarish va xabarnomalar uchun servis
service BotAdminService {
    // To'lov muvaffaqiyatli bo'lganda Go servisi tomonidan chaqiriladi
    rpc NotifyPaymentSuccess(NotifyPaymentSuccessRequest) returns (google.protobuf.Empty);
}

message NotifyPaymentSuccessRequest {
    string student_name = 1;
    string branch_name = 2;
    string group_name = 3;
    int64 amount = 4;
    string payment_time = 5; // Format: "YYYY-MM-DD HH:MM:SS"
}